tests/kuttl/test-suites/default/deps/keystone.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: core.openstack.org/v1beta1
```

Comment:
why do we need this file? Don't we already enable keystone in the controlplane defined in infra.yaml?

Reply:
that's just to establish the minimal required parameters for keystone by kustomize and leave the infra.yaml as clean as possible. In any case i can merge it into the infra.yaml